pom.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.qameta.allure.examples</groupId>
+    <artifactId>allure-cucumber7-junit-platform-maven</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+        <cucumber.version>7.14.0</cucumber.version>
+        <allure.version>2.24.0</allure.version>
+        <aspectj.version>1.9.19</aspectj.version>
+        <junit-platform.version>1.10.0</junit-platform.version>
+
+    </properties>
+
```

Comment:
add 

```xml
    <dependencyManagement>
        <dependencies>
            <dependency>
                <groupId>io.qameta.allure</groupId>
                <artifactId>allure-bom</artifactId>
                <version>${allure.version}</version>
            </dependency>
            <dependency>
                <groupId>org.junit</groupId>
                <artifactId>junit-bom</artifactId>
                <version>{{ junitBom }}</version>
            </dependency>
        </dependencies>
    </dependencyManagement>
```

And then omit the versions from managed dependencies.

Reply:
At first I took this remark literally and failed even after studying the issue.
Please check out the changes in the new commit and let me know if my proposed approach is different from what you had in mind.

Reply:
In particular, I dont understand how `{{ junitBom }}` works

Reply:
> In particular, I still dont understand how `{{ junitBom }}` works or should work. Mvn (./mvnw) throws exceptions every time

that is placeholder that need to be replaced ;)